development/shared/neutron-ext-net-ksv3

```diff
@@ -92,7 +95,7 @@ if __name__ == '__main__':
         if opts.network_type == 'vxlan':
             network_msg['network']['provider:segmentation_id'] = 1234
         elif opts.network_type == 'vlan':
-            network_msg['network']['provider:segmentation_id'] = 2
+            network_msg['network']['provider:segmentation_id'] = opts.vlan_id 
```

Comment:
Update the tools in this repository:
https://github.com/openstack-charmers/openstack-charm-testing